build-scripts/build-auth-rpm

```diff
@@ -170,7 +170,8 @@ make install DESTDIR=%{buildroot}
 
 %{__rm} -f %{buildroot}%{_libdir}/%{name}/*.la
 %{__install} -p -D -m 0755 pdns/pdns %{buildroot}%{_initrddir}/pdns
-%{__mv} %{buildroot}%{_sysconfdir}/%{name}/pdns.conf{-dist,}
+%{buildroot}/usr/bin/pdns_server --config > %{buildroot}%{_sysconfdir}/%{name}/pdns.conf
+%{__rm} %{buildroot}%{_sysconfdir}/%{name}/pdns.conf-dist
```

Comment:
> is the change from `.conf-dist` to `.conf` intentional?

Yes, `make install` puts the `pdns.conf-dist` in place. Before we `mv`'ed it. But don't need it anymore, so it has to go.

> also, recommend putting `--no-config` in front of `--config` to avoid (launch) contamination

Will do